package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "blue-blocker",
-	"version": "0.4.9",
+	"version": "0.4.10",
```

Comment:
why not go to 0.5.0 next?

Reply:
I guess I don't think it's a big enough change for a minor version, and semver lets you have non-single digit parts so why not?

Reply:
well I was wondering if maybe you might want to include some other changes as well?

Reply:
eh whatever